.appveyor.yml

```diff
@@ -11,6 +11,7 @@ environment:
 
 cache:
     - '%CYG_CACHE%'
+    - C:\projects\qfs\build # appveyor is super slow -- cache the build so we only build diffs
```

Comment:
nit:
CYG_CACHE: C:\cygwin\var\cache\setup... --> CYGROOT\var\cache...
CYG_BASH: C:\cygwin\bin\bash  --> CYGROOT\bin\bash...
Would that work?